### DISCUSSION TITLE
I am signing into the wrong account

### DISCUSSION THREAD

**Discussion by RATIU5** [vercel/community#6466] [Account Issue] [answered]
### Question

I am currently trying to sign into my personal account but end up signed into my work account. Is this a bug?
I tried to sign into my personal account through GitHub, which is where it was created from. My work account was created from an email. I can sign into my work account with an email verification fine. I cannot sign into my personal account through GitHub anymore as it thinks I'm on my work account. I can't find a way to access or manage all my personal projects. Any help is appreciated.

**Comment by amyegan** [comment]
Hey @RATIU5. Is your work account email the same as your GitHub primary email? Or did you connect your GitHub account to the your work Vercel account?

**Reply by RATIU5** [reply]
No, they are different emails. I created a new account with a different email (work email) for my work account. I have a personal email attached to my GitHub account.

**Comment by RATIU5** [comment]
This is still a problem. I cannot log into my former account.

**Reply by amyegan** [reply, marked as answer]
If you connected GitHub to your work account, then the connection to your personal account would have been removed. Since each account uses a different email address, your personal account should be accessible [using your personal email](https://vercel.com/login/email) even though you can't log into it with GitHub anymore. 

We can't safely discuss account information in public due to privacy and security concerns so I can't get enough info to fully solve login issues from here. If you continue to have trouble logging in, please email [registration@vercel.com](mailto:registration@vercel.com) to open a ticket with the team that can help with login issues. 

Be sure to let them know the email addresses used for your accounts and how you use each account (i.e. one is for work and the other is for personal projects). That will help them locate and verify both accounts. 🙏

https://vercel.com/guides/how-to-get-vercel-support#direct-channels
